internal/provider/data_source_gitlab_repository_tree_test.go

```diff
@@ -0,0 +1,49 @@
+//go:build acceptance
+// +build acceptance
+
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceGitlabRepositoryTree_basic(t *testing.T) {
+	testProject := testAccCreateProject(t)
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+					resource "gitlab_repository_file" "foo" {
```

Comment:
It would also be nice to just add some files in the `PreConfig` hook instead of having them in the tf config here.

Reply:
Should I just use an API call or create a new `testAccCreateProjectFile` function for this purpose

Reply:
hmmm, let's go with the function. I think we can also use it in the tests for the `gitlab_repository_file` data source at one point :)

Reply:
All done